hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -460,41 +465,38 @@ class BatchClient:
         self._headers = h
 
     async def _get(self, path, params=None):
-        response = await retry_request(
+        return await retry_request(
```

Comment:
What was your rationale for not returning the json directly?

Reply:
The json might not always be there, at least for other verbs (in particular, cancel is a post that returns no json).  I changed get so the interface was consistent.

Reply:
Err, no, cancel is a patch.  jobs/create is post.